### ISSUE TITLE
Can't connect to remote host if the SSH key has no password

### ISSUE THREAD

**Issue by mr-mmmmore** [Sequel-Ace/Sequel-Ace#232]
- Sequel Ace Version: 2.1.2
- macOS Version: 10.14.6
- MySQL Version:  Ver 15.1 Distrib 10.2.27-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2

**Description**
I am trying to connect to a remote DB through SSH, using a key that has no password defined. I get a window that asks for a password for \<my_host_user_name\>@\<my-host.com\>. If I type nothing and submit, the connection fails.
![image](https://user-images.githubusercontent.com/14160945/87518247-51e07700-c680-11ea-845c-1d98270d4703.png)

**Steps To Reproduce**
1. Install a key with no password on the remote host
2. Configure a SSH connection to this host using this key
3. Try to connect
4. The password window shows, type nothing and submit
5. The password window shows again

**Expected Behaviour**
Either the password window shouldn't show at all, or submitting an empty password should result in a successful connection.

**Is Issue Present in [Latest Beta](https://github.com/Sequel-Ace/Sequel-Ace/releases)?**
The issue has been there since version 2.1.0 when I installed the app for the first time. I haven't tried with the 2.1.3-rc1 version since it doesn't seem to bring anything new to this problem.

**Additional Context**
- I can ssh to the host in a terminal  using that same key.
- I am using the same exact config for this remote host that works in Sequel Pro.

Here are the settings:
![image](https://user-images.githubusercontent.com/14160945/87517230-ddf19f00-c67e-11ea-85a7-d73ea2a02ea3.png)

**Comment by Jason-Morcos** [comment]
@mr-mmmmore I believe that prompt is not asking you for your ssh key password - it's asking your for your MySQL password on the remote database.

**Comment by mr-mmmmore** [comment]
I don't think so, the prompt says:

> \<my_host_user_name\>@\<my-host.com\> password's

which refers to the user on the remote host, not the mysql user (which is `my_db_user_name`in the screen capture).

Besides, I have also tried typing the mysql password in there and it doesn't work neither.

**Comment by Jason-Morcos** [comment]
Hmmm. I just tested and I am able to successfully connect with an ssh key with no password without being prompted at all.
Please try deselecting and reselecting your ssh key. And verify that you can indeed ssh into your remote host with your ssh key outside of Sequel Ace. Is it possible your ssh key is not actually accepted/known by the host you're trying to connect to? It looks possible that your id_rsa key is being declined by the remote host.

**Comment by mr-mmmmore** [comment]
I have two ssh keys in my .ssh folder, and it seems Sequel Pro from which I took the connection settings is displaying the bad one: in Sequel Pro, the key displayed in the config is the default one (id_rsa), but the one actually installed on the server is my other SSH key. So I don't know how Sequel Pro manages to connect, but whatever, I managed to connect using the good key in Sequel Ace.

Thank you and sorry for the bug report.
